notification-svc/models/sendEmail.ts

```diff
@@ -16,7 +16,10 @@ function sendEmail(msg: string) {
             }],
             "Subject": `${msg}`,
             "TextPart": "My first Mailjet email",
-            "HTMLPart": "<h3>Dear patient, welcome to <p>MedCal</p>!</h3><br />We wish you get better asap!"
+            "HTMLPart": `<h3>Dear patient, welcome to MedCal!</h3><br />
+            We wish you get better asap!<br /><br />
+            <a href="http://localhost:3001/notification/rescheduling-confirm" style="background-color: #4CAF50; color: white; padding: 10px 20px; text-align: center; text-decoration: none; display: inline-block; border-radius: 5px;">Confirm</a><br /><br />
```

Comment:
Shouldn't it be redirecting users to a webpage? From where are we calling the API endpoint to make a crud update? If so, i think, should include name, last name, and emails as query parameters.
P.S. Or is it just for testing purposes?

Reply:
yes, the confirmation logic is not complete here, but with this setup Deepu will be able to run his part with some meaningful output (email being sent), and also I was not sure in what shape his data will be coming, so we will adjust this part later